### ISSUE TITLE
clippy output frustratingly trim messages

### ISSUE THREAD

**Issue by dlukeomalley** [returntocorp/bento#155]
Reported by both @ievans and @dmontagu:

---

In particular, the character trimming into ... is very annoying, as I can't read the full output of some check text:

` 12:13 error   Loop control variable 'i' not used within the loop body. If this is intended, start the name with... r2c.flake8     B007`

_Originally posted by @ievans in https://github.com/returntocorp/bento/issues/101#issuecomment-546134449_

---

This case is particularly frustrating:
```
306:13 error   Loop control variable 'label' not used within the loop body. If this is intended, start the name ... r2c.flake8     B007 https://lintlyci.github.io/Flake8Rules/rules/B007.html
```

I would be happy to fix it by following the instructions, but it is cut off before the end. So I try to go to the link for more details, and it is broken. (I'm assuming it's saying start with a `_`, but it would still be nice if there was a way to see the full description.)

_Originally posted by @dmontagu in https://github.com/returntocorp/bento/issues/152#issuecomment-550507842_

**Comment by dlukeomalley** [comment]
Because multiple users have not reported this issue, I'm promoting it to a `bug` given the level of frustration it is causing.

**Comment by ievans** [comment]
We should fix this ASAP, previously reported here: https://github.com/returntocorp/bento/issues/101#issuecomment-542367034_

**Comment by dlukeomalley** [comment]
Downscoping to Clippy formatting since it's our default

**Comment by nbrahms** [comment]
Fixed for 0.8
